### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alidehkhodaei**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # test
```
Comment (alidehkhodaei) [inline README.md:1]: Hello 1

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
...
+1
```
Comment (alidehkhodaei) [inline README.md:2]: 2

Comment (alidehkhodaei) [inline README.md:2]: Remove

**Review by alidehkhodaei — COMMENTED** [review]
Removie

**Review by asimpledeveloper — CHANGES_REQUESTED** [review]
fdfdf

**Pull request merged**